src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -59,7 +59,8 @@
    :csrdir                   schema/Str
    :ruby-load-path           [schema/Str]
    :signeddir                schema/Str
-   :serial                   schema/Str})
+   :serial                   schema/Str
+   :keylength                schema/Int})
```

Comment:
During development I found it very useful to have these settings in alphabetical order.  We might as well stick to it here and put it between `:csrdir` and `:ruby-load-path`.

Same goes for the `MasterSettings` if/when we add it there too.